test/support/lit.cfg.in

```diff
@@ -0,0 +1,21 @@
+import site, os
+site.addsitedir(os.path.join('@CMAKE_SOURCE_DIR@', 'test', 'support'))
+import testformat
+
+config.name = 'Beman project test suite'
+config.test_format = testformat.CxxStandardLibraryTest()
+config.test_exec_root = '@CMAKE_CURRENT_BINARY_DIR@'
+config.test_source_root = '@CMAKE_CURRENT_SOURCE_DIR@'
+flags = [
+    '-std=c++20',
+    '-isysroot @CMAKE_OSX_SYSROOT@' if '@CMAKE_OSX_SYSROOT@' else '',
+    '-isystem {}'.format(os.path.join('@CMAKE_SOURCE_DIR@', 'src', 'example')),
+    '@CMAKE_CXX_FLAGS@'
+]
```

Comment:
When the project under test is more complex, I expect that there will be transitive usage requirements that need to get passed to the compilation flags here. Do those need to be repeated manually or is there a way to query cmake?

Reply:
Another thing that needs to be verified is cross-compilation support.